Error when not using zip feature

# Problem
The _kml_ crate does not compile without the _zip_ feature.
```
error[E0433]: failed to resolve: use of undeclared crate or module `zip`
  --> kml/src/errors.rs:34:22
   |
34 |     ZipError(#[from] zip::result::ZipError),
   |                      ^^^ use of undeclared crate or module `zip`
```
# Steps to Reproduce
Create a new crate with cargo and add `kml = { version = "0.4.1", default-features = false }` to the dependencies.
`cargo check` now fails.
# Possible Solution
Adding the line `#[cfg(feature = "zip")]` before `#[error("ZIP error: {0}")]` in `errors.rs` fixes the issue for me.

--- a/src/errors.rs
+++ b/src/errors.rs
@@ -30,6 +30,7 @@ pub enum Error {
     InvalidListItemType(String),
     #[error("IO error: {0}")]
     IoError(#[from] std::io::Error),
+    #[cfg(feature = "zip")]
     #[error("ZIP error: {0}")]
     ZipError(#[from] zip::result::ZipError),
     #[error("Invalid units: {0}")]

